templates/caddy.service

```diff
@@ -68,6 +68,17 @@ NoNewPrivileges=true
 Environment={{ key }}={{ value }}
 {% endfor %}
 
+{% endif %}
+
+{% endif %}
+{% if caddy_environment_files|length %}
+
+; Additional environment files:
+
+{% for path in caddy_environment_files %}
+EnvironmentFile=-path
```

Comment:
Shouldn't `path` be templated?

Reply:
> The argument passed should be an absolute filename or wildcard expression, optionally prefixed with "-", which indicates that if the file does not exist, it will not be read and no error or warning message is logged. 

Is there a reason you want to prefix the path with a dash? I think an error or warning would be appropriate here if the file is not found?